lib/ansible/modules/hostname.py

```diff
@@ -803,6 +803,12 @@ class CentOSHostname(Hostname):
     strategy_class = RedHatStrategy
 
 
+class AnolisOSHostname(Hostname):
```

Comment:
@geliwei Thanks for the PR. I will wait for https://github.com/ansible/ansible/pull/74797 to get merged first. Then we can merge this.

Apart from that, please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
I didn't file a bug for this since it's a minor change, hope that will be OK.

Reply:
Yes that is OK. Just include the PR link in the changelog.